{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "datePublic": "2020-04-20T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "An issue was discovered in OpenVPN 2.4.x before 2.4.9. An attacker can inject a data channel v2 (P_DATA_V2) packet using a victim's peer-id. Normally such packets are dropped, but if this packet arrives before the data channel crypto parameters have been initialized, the victim's connection will be dropped. This requires careful timing due to the small time window (usually within a few seconds) between the victim client connection starting and the server PUSH_REPLY response back to the client. This attack will only work if Negotiable Cipher Parameters (NCP) is in use."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2022-05-03T13:06:08",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://github.com/OpenVPN/openvpn/commit/37bc691e7d26ea4eb61a8a434ebd7a9ae76225ab"
                },
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://security-tracker.debian.org/tracker/CVE-2020-11810"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://bugzilla.suse.com/show_bug.cgi?id=1169925"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://community.openvpn.net/openvpn/ticket/1272"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://patchwork.openvpn.net/patch/1079/"
                },
                {
                    "name": "FEDORA-2020-c1cb4ebcd9",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_FEDORA"
                    ],
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/FGHHV4YZANZW45KZTJJGVGPFMSXYRCKZ/"
                },
                {
                    "name": "FEDORA-2020-969414e05b",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_FEDORA"
                    ],
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/JII7RYYYRBPQNEGGVSOXCM7JUZ43T3VH/"
                },
                {
                    "name": "[debian-lts-announce] 20220503 [SECURITY] [DLA 2992-1] openvpn security update",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "https://lists.debian.org/debian-lts-announce/2022/05/msg00002.html"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2020-11810",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "An issue was discovered in OpenVPN 2.4.x before 2.4.9. An attacker can inject a data channel v2 (P_DATA_V2) packet using a victim's peer-id. Normally such packets are dropped, but if this packet arrives before the data channel crypto parameters have been initialized, the victim's connection will be dropped. This requires careful timing due to the small time window (usually within a few seconds) between the victim client connection starting and the server PUSH_REPLY response back to the client. This attack will only work if Negotiable Cipher Parameters (NCP) is in use."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://github.com/OpenVPN/openvpn/commit/37bc691e7d26ea4eb61a8a434ebd7a9ae76225ab",
                            "refsource": "CONFIRM",
                            "url": "https://github.com/OpenVPN/openvpn/commit/37bc691e7d26ea4eb61a8a434ebd7a9ae76225ab"
                        },
                        {
                            "name": "https://security-tracker.debian.org/tracker/CVE-2020-11810",
                            "refsource": "MISC",
                            "url": "https://security-tracker.debian.org/tracker/CVE-2020-11810"
                        },
                        {
                            "name": "https://bugzilla.suse.com/show_bug.cgi?id=1169925",
                            "refsource": "CONFIRM",
                            "url": "https://bugzilla.suse.com/show_bug.cgi?id=1169925"
                        },
                        {
                            "name": "https://community.openvpn.net/openvpn/ticket/1272",
                            "refsource": "CONFIRM",
                            "url": "https://community.openvpn.net/openvpn/ticket/1272"
                        },
                        {
                            "name": "https://patchwork.openvpn.net/patch/1079/",
                            "refsource": "CONFIRM",
                            "url": "https://patchwork.openvpn.net/patch/1079/"
                        },
                        {
                            "name": "FEDORA-2020-c1cb4ebcd9",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FGHHV4YZANZW45KZTJJGVGPFMSXYRCKZ/"
                        },
                        {
                            "name": "FEDORA-2020-969414e05b",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/JII7RYYYRBPQNEGGVSOXCM7JUZ43T3VH/"
                        },
                        {
                            "name": "[debian-lts-announce] 20220503 [SECURITY] [DLA 2992-1] openvpn security update",
                            "refsource": "MLIST",
                            "url": "https://lists.debian.org/debian-lts-announce/2022/05/msg00002.html"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2020-11810",
        "datePublished": "2020-04-27T14:47:01",
        "dateReserved": "2020-04-15T00:00:00",
        "dateUpdated": "2022-05-03T13:06:08",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}